styles/avatar.css

```diff
@@ -0,0 +1,89 @@
+.main-container-components {
+  margin-left: 15rem;
+}
+
+.container-text {
+  width: 15rem;
+}
+
+.heading {
+  margin-left: 1rem;
+  font-size: 2.5rem;
+  font-weight: 500;
+}
+
+.description {
+  padding: 1rem;
+  font-size: 1.2rem;
+}
+
+.heading-avatar {
+  font-size: 2rem;
+}
+
+.description-avatar {
+  font-size: 1.1rem;
+}
+
+.avatar-container {
+  background-color: var(--body-background-color);
+  max-width: 100%;
+  height: 10rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  flex-wrap: wrap;
+}
+
+.avatar {
```

Comment:
Avoid using absolute units, try to use relative ones. It will help you in making pages responsive.

Reply:
Done!! added units in REM